specs/challenge.md

```diff
@@ -216,16 +216,90 @@ until time `t`".
 
 The following verification process applies to output version 1 and later:
 
-The `_pair[4]` and `_pair[5]` contain the [public input](./zkevm-prover.md#zkevm-proof), which must be processed before
+Since Colosseum verifies public input along with zkevm-proof, challengers should calculate as below 
+and enclose the public input to the `proveFault` transaction.
```

Comment:
```suggestion
Since Colosseum verifies public input along with [zkevm-proof](./zkevm-prover.md#zkevm-proofs), challengers should
calculate as below and enclose the public input to the `proveFault` transaction.
```

Reply:
applied